db/migrate/20181005182645_add_tenant_and_tenant_id_to_all_tables.rb

```diff
@@ -0,0 +1,20 @@
+class AddTenantAndTenantIdToAllTables < ActiveRecord::Migration[5.0]
+  create_table "tenants", id: :bigserial, force: :cascade do |t|
```

Comment:
Shouldn't this be in the `change` method?

Reply:
Doh, you're right. I threw it in last minute. I'll take care of that and also add the model.